lobby/src/main/java/org/triplea/lobby/server/login/forgot/password/create/PasswordEmailSender.java

```diff
@@ -0,0 +1,57 @@
+package org.triplea.lobby.server.login.forgot.password.create;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Properties;
+import java.util.function.BiConsumer;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import org.triplea.lobby.server.EnvironmentVariable;
+
+/** Sends a temporary password to a target user. */
+class PasswordEmailSender implements BiConsumer<String, String> {
+  private static final String FROM = "no-reply@triplea-game.org";
```

Comment:
Is this actually a valid email address?
I mean SMTP allows any address, but that might cause the mails to end up in the spam filter.

Reply:
AFAIK it is valid; Sending a test mail, the mail did not land in spam folder. Anything specific to say it's not valid?

Reply:
Let me clarify:
I wasn't aware that triplea-game.org is a valid mail address.
AFAIK when the source email server and the from email address don't match, common email servers just put them directly into the spam folder, but I'm not sure how accurate this information is.

Basically I'm wondering what exactly the java mail API does. Is it just talking to some SMTP server somewhere, or is it actually an SMTP server implementation itself that just sends mails directly to other POP3 or IMAP servers?

Reply:
> AFAIK when the source email server and the from email address don't match, common email servers just put them directly into the spam folder, but I'm not sure how accurate this information is

That really depends on the email provider. I wonder how accurate that rule works as it's possible to have email relays. Mail from `no-replay@amazon.com` for example I would confidently bet does not come from a `amazon.com` server. That mail would come from some random ec2 instance that would probably have `amazon.com` as a domain suffix, but it would not be `amazon.com` proper.


---------

> Is it just talking to some SMTP server somewhere

Basically. It can be configured to do more.